Latex/samplepaper.tex

```diff
@@ -0,0 +1,288 @@
+Dieser nimmt Sensordaten vom Mqtt-Sensordaten-Server entgegen.
+Damit diese Daten in das richtige Schema der Datenbank persistiert werden können, bietet der Service die Möglichkeit sich darauf einzutragen (subscribe).
+Eingetragene Beobachter (Observer) werden über eingehende Sensordaten informiert.
+Für die Beobachter wird eine Schnittstelle (Interface) bereit gestellt.
+Diese Schnittstelle wird von einer konkreten Klasse implementiert, dem SensorDataWorker.
+Dieser SensorDataWorker filtert die Daten, ruft den DatenbankService auf und übergibt die relevanten Daten.
+Durch dieses Beobachter-Muster ist es möglich weitere Daten über separate Beobachter zu verarbeiten, ohne das bisherige Beobachter angepasst werden müssen.
+\end{description}
+
+
+\subsection{Begründung}
+Der Vorteil des Kompositum-Musters ist, dass der Sensorenbereich (SensorArea) beliebig erweitert werden kann und es dabei zweitrangig ist um was die Shops beziehungsweise andere Komponenten erweitert werden. 
+Ein Nachteil dabei ist, dass die Tiefe ebenfalls beliebig wächst und es so zu einer sehr unübersichtlichen Datenstruktur kommen kann.
+Eine Alternative beziehungsweise Erweiterung wäre das Fabrikmethode-Muster. 
+Dieses Muster erlaubt es uns auf Grundlage der abstrakten Klasse SensorArea weitere Bereiche zu implementieren, ohne dass davon andere Services betroffen sind. 
+Man könnte also Shop oder Einkaufszentrum erzeugen, würde dabei aber die Möglichkeit der Schachtelung verlieren. 
+Um beide Entwurfsmuster möglichst effektiv zu verwenden, wäre eine Kombination denkbar.
+\\
+
+
+Das Beobachter-Muster wurde gewählt, da es eingehende Signale von den Sensoren beziehungsweise Sensornachrichten automatisch verarbeitet und speichert.
+Dieses Muster könnte durch das Zustands-Muster erweitert, aber nicht ersetzt werden.
+Das Zustands-Muster würde in der konkreten Implementierung ebenfalls auf eingehende Nachrichten warten / reagieren, es könnte beispielsweise als ein Buffer agieren welcher Nachrichten sammelt und erst auf Anfrage Daten weiter leitet / speichert.
+Dieses Verhalten würde das Beobachter-Muster eher erweitern oder imitieren und ist daher ungeeignet in diesem Kontext.
+
+
+\newpage
+\section{Beschreibung der Kernfunktionalität}
+Eine der Kernfunktionalitäten ist es, die Auslastung von Shops zu ermitteln und mit dieser Information weiter zu arbeiten.
+Daher stellen wir in diesem Abschnitt die Funktionalität zur Ermittlung der Auslastung eines Einkaufzentrums (ShoppingCenter) dar.
+
+\includegraphics[width=\linewidth]{images/Sequenzdiagramm}
+In dem Sequenzdiagramm beschriebenen Beispiel, möchte ein Client die Auslastung eines bestimmten Shops haben und ruft daher die Methode calculateUsage auf dem SmartShopService auf.
+\\
+
+\textbf{SmartShopService}: In diesem Beispiel wird davon ausgegangen, dass es bei der Auflösung einer bestimmten ShopOID nicht zu einem Fehler kommt sondern die Auflösung immer erfolgreich ist.
```

Comment:
Den Effekt, den du hier versuchst zu erreichen, kannst du auch über \paragraph erreichen, das ist dann sogar noch etwas sauberer in latex.